sites/main-site/src/content/hackathon-projects/hackathon-march-2025/single-cell-atlas.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Single Cell Expression Atlas to nf-core"
+category: pipelines
+slack: ""
+intro_video: ""
+image: "/assets/images/events/2025/hackathon-march/single-cell-atlas.jpg"
+image_alt: Logo by Expression Atlas team, EMBL-EBI
```

Comment:
please describe the image in the alt-text: https://accessibility.huit.harvard.edu/describe-content-images

Reply:
done [3cdfd24](https://github.com/nf-core/website/pull/3065/commits/3cdfd24036efba0a794f73f1ea8308e458cfcd83)